cinema_service/settings.py

```diff
@@ -134,7 +138,7 @@
 STATIC_URL = "static/"
 
 MEDIA_URL = "/media/"
-MEDIA_ROOT = BASE_DIR / "media"
+MEDIA_ROOT = "/files/media"
```

Comment:
The `MEDIA_ROOT` should be an absolute path. Ensure that the path `/files/media` is correctly set up in your environment.